tests/Unit/Command/FeedItemListTest.php

```diff
@@ -0,0 +1,158 @@
+<?php
+/**
+ * @author Sean Molenaar <sean@seanmolenaar.eu>
+ *
+ * @license AGPL-3.0
+ *
+ * This code is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License, version 3,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License, version 3,
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>
+ *
+ */
+
+namespace OCA\News\Tests\Unit\Command;
+
+use OCA\News\Command\Debug\FeedItemList;
+use OCA\News\Command\Updater\UpdateFeed;
+use OCA\News\Service\ItemServiceV2;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class FeedItemListTest extends TestCase
+{
+    /** @var MockObject|ItemServiceV2 */
+    protected $service;
+    /** @var MockObject|InputInterface */
+    protected $consoleInput;
+    /** @var MockObject|OutputInterface */
+    protected $consoleOutput;
+
+    /** @var UpdateFeed */
+    protected $command;
```

Comment:
The type hint is probably wrong  .. shouldn't it be `FeedItemList` instead?
An then the `use OCA\News\Command\Updater\UpdateFeed;` is also unnecessary.
(Applies to all new tests)